test/cors.test.js

```diff
@@ -138,13 +129,11 @@ test('Should create a options wildcard (with prefix)', t => {
     delete res.headers.date
     t.strictEqual(res.statusCode, 204)
     t.strictEqual(res.payload, '')
-    t.deepEqual({
+    t.match(res.headers, {
       'access-control-allow-origin': '*',
       'access-control-allow-methods': 'GET,HEAD,PUT,PATCH,POST,DELETE',
-      vary: 'Access-Control-Request-Headers',
-      'content-length': '0',
```

Comment:
This content-length should remain since it is added by
https://github.com/fastify/fastify-cors/blob/0365bdbe79ce03227360df093d4d6c596e506fc3/index.js#L85-L87